### PR TITLE
Bugfixes/join the squad page

### PR DESCRIPTION
**Pull request by artofmayhem**
Removal of JoinTheSquad fully

This component was deprecated and not to be used moving forward. 

Simple deletion of: 

Folder : JoinTheSquad
stylesheet: JoinTheSquad.less
removal of imports and route JoinTheSquad from index.js

Simple pimple and done.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -243,13 +242,6 @@ function App() {
...
           )}
         />
-        <ProtectedRoute
-          exact
-          path="/child/join"
-          component={() => (
-            <JoinTheSquad LoadingComponent={ChildLoadingComponent} />
-          )}
-        />
         <ProtectedRoute
```
Comment (MichaelMcFaddenjr) [inline src/index.js:245]: Nice work making sure you got everything out!!

**Review by MichaelMcFaddenjr — DISMISSED** [review]

**Review by rhiannonstanford — DISMISSED** [review]
Good work catching that this page did not have a figma and the component was deprecated.  Nice job!

**Review by milklot — DISMISSED** [review]
Good job, let's keep this App cleaner !

**Review by rhiannonstanford — APPROVED** [review]
Excellent add!

**Review by mollybee — APPROVED** [review]
Thorough job in extracting the JoinTheSquad.

**Review by milklot — APPROVED** [review]
One more step in the bug fix, nice job

**Review by MichaelMcFaddenjr — APPROVED** [review]

File: src/styles/less/JoinTheSquad.less [hunk 2]
```diff
@@ -1,169 +0,0 @@
-@import '~antd/dist/antd.less';
```
Comment (Julianisrel) [inline src/styles/less/JoinTheSquad.less:1]: Great job cleaning up the app!

**Pull request merged**